src/components/RichTextEditor/helpers/isLocalValueEmpty.ts

```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { SlateToMarkdown, Transform } from '../serializer';
+import { TreeOfNodes } from '../types';
+
+export const isLocalValueEmpty = (value: TreeOfNodes): Promise<boolean> =>
+    new Promise((resolve) => {
+        const result = Transform.use<TreeOfNodes, string>(new SlateToMarkdown()).process(value);
+        resolve(result.length === 0);
+    });
```

Comment:
Why `Promise`?
```suggestion
export const isLocalValueEmpty = (value: TreeOfNodes): boolean => {
    const result = Transform.use<TreeOfNodes, string>(new SlateToMarkdown()).process(value);
    return result.length === 0;
}
```

Reply:
If `process` is not async then wrapping it with a `Promise` has not effect

Reply:
But hen you call it it has effect because you are using it like 
```
isLocalValueEmpty().then() ... 
```
and it wont stop the process nevertheless how long it makes the transformation.
Or do I understand it wrong?